src/main/java/peoplesoft/model/person/Email.java

```diff
@@ -30,11 +30,11 @@
 
     private static final String SPECIAL_CHARACTERS = "+_.-";
     private static final String SPECIAL_CHARACTERS_NO_PERIOD = "+_-";
-    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain "
+    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format name@domain "
             + "and adhere to the following constraints:\n"
-            + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
+            + "1. The name should only contain alphanumeric characters and these special characters, excluding "
             + "the parentheses, (" + SPECIAL_CHARACTERS + "). The local-part may not start or end with any special "
-            + "characters.\n"
+            + "characters. Scroll down to read more.\n"
```

Comment:
not sure about leaving this in the constraints message (+ will different screen sizes or font renderers or w/e cause it to *not* appear in the right place?)

but then again we're a little short on time, so w/e